app/src/views/terms.pug

```diff
@@ -142,11 +142,11 @@ block main
 
       7.3. GovTech reserves all rights:
 
-      7.3.1.  to disable any links to, or frames of, any materials which are unauthorised (including without limitation materials which imply endorsement by or association or affiliation with GovTech, materials containing inappropriate, profane, defamatory, infringing, obscene, indecent or unlawful topics, names, or information that violates any written law, any applicable intellectual property, proprietary, privacy or publicity rights); and
+      7.3.1. to disable any links to, or frames of, any materials which are unauthorised (including without limitation materials which imply endorsement by or association or affiliation with GovTech, materials containing inappropriate, profane, defamatory, infringing, obscene, indecent or unlawful topics, names, or information that violates any written law, any applicable intellectual property, proprietary, privacy or publicity rights); and
```

Comment:
Is there anything different here? Not sure why it is showing up as a change.

Reply:
It's because of the removal of the superfluous space at the start of the Claude.